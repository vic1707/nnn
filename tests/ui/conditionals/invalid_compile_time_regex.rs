#![expect(unused_imports, reason = "Not what we're testing.")]
//! Is compile time error if feature `regex_validation` is enabled
//! Else it will compile but fail at runtime.
use nnn::nnn;

#[nnn(validators(regex = r#"(\d+"#))]
struct InvalidRegex(String);

fn main() {}
