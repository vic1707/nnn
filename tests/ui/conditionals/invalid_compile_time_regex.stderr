error: Incorrect Regex regex parse error:
           (\d+
           ^
       error: unclosed group
 --> tests/ui/conditionals/invalid_compile_time_regex.rs:6:26
  |
6 | #[nnn(validators(regex = r#"(\d+"#))]
  |                          ^^^^^^^^^
